vaadin-spring/src/main/java/com/vaadin/flow/spring/VaadinServletContextInitializer.java

```diff
@@ -631,9 +632,12 @@ public void onStartup(ServletContext servletContext)
 
     private Collection<String> getErrorParameterPackages() {
         return Stream
-                .concat(Stream
-                        .of(HasErrorParameter.class.getPackage().getName()),
-                        getDefaultPackages().stream())
+                .concat(Stream.of(
+                        SpringRouteNotFoundError.class.getPackage().getName()),
```

Comment:
Why don't you add this new package like this:

```
Stream.of(HasErrorParameter.class.getPackage().getName(), SpringRouteNotFoundError.class.getPackage().getName())
```
?

Reply:
Thanks, that's cleaner (I forgot that `Stream.of` is variadic).